js/bittrex.js

```diff
@@ -681,6 +688,89 @@ module.exports = class bittrex extends Exchange {
         return this.parseTrades (response, market, since, limit);
     }
 
+    async fetchTradingFee (symbol, params = {}) {
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        const request = {
+            'marketSymbol': market['id'],
+        };
+        const response = await this.privateGetAccountFeesTradingMarketSymbol (this.extend (request, params));
+        //
+        // {"marketSymbol":"1INCH-ETH","makerRate":"0.00750000","takerRate":"0.00750000"}
+        //
+        return this.parseTradingFee (response);
+        // {
+        //     info: {
+        //       marketSymbol: '1INCH-ETH',
+        //       makerRate: '0.00750000',
+        //       takerRate: '0.00750000'
+        //     },
+        //     symbol: '1INCH/ETH',
+        //     maker: 0.0075,
+        //     taker: 0.0075
+        // }
```

Comment:
Please, always pass the second argument (`market` or `currency`) if it's accessible – to the parsing method like `parseTradingFee`, `parseOrder`, `parseTrade`, etc.
Also, we don't have to document the unified response here, please remove the comment from lines 702-711.
We only need to document the implicit endpoint response as you did in lines 698-700.
Also, if possible, please follow this style for response samples for better readability:

```JavaScript
        const response = await this.privateGetAccountFeesTradingMarketSymbol (this.extend (request, params));
        //
        //     {
        //         "marketSymbol":"1INCH-ETH",
        //         "makerRate":"0.00750000",
        //         "takerRate":"0.00750000"
        //     }
        //
```